patterns/1-initial/introducing-metrics-in-innersource.md

```diff
@@ -62,7 +62,7 @@ The list below consolidates some metrics candidates, grouped by business goals.
 | Knowledge sharing | # of clone or forks from external guests<br># of visits from external guests |
 | Creating reusable software | # of submodule usage<br># of code reuse across projects<br># of clone or forks from external guests |
 | Increase development speed | Time to market<br>Mean-time-to-review |
-| Employee satisfaction & talent retention |  |
+| Employee satisfaction & talent retention | Employee satisfaction & talent retention | eNPS score | attrition rate | 
```

Comment:
```suggestion
| Employee satisfaction & talent retention | eNPS score<br>attrition rate | 
```

@Trueblueaddie I suspect you wanted to list both "eNPS score" as well as "attrition rate" as metric candidates for "Employee satisfaction & talent retention"?

I am reasonably sure, so I will go ahead and fix the formatting and merge this PR.